nwb_conversion_tools/utils.py

```diff
@@ -7,6 +7,10 @@
 import pynwb
 
 
+def get_schema_data(in_data, data_schema):
+    return {k: in_data.get(k, None) for k in data_schema['properties'].keys()}
```

Comment:
```suggestion
    return {k: in_data[k] for k in data_schema['properties'] if k in in_data}
```

How about this, which doesn't auto-fill `None` for missing data

Reply:
Yeah, that's probably for the best. If there are missing input arguments or conversion options, if they are (a) optional, the downstream functions will use their own defaults, which aren't always `None`, or (b) required, best to not attempt to auto-population them anyway. Fixing now.